include/wifi_logger.h

```diff
@@ -27,6 +31,11 @@ void generate_log_message(esp_log_level_t level, const char *TAG, int line, cons
 int system_log_message_route(const char* fmt, va_list tag);
 void start_wifi_logger(void);
 void wifi_logger();
+bool is_connected(void* handle_t);
```

Comment:
Can you use int instead of bool

Reply:
is_connected is just for checking if connected or not since status is saved in errno by esp